samples/FakeItEasy.Examples/CustomArgumentValidators.cs

```diff
@@ -1,12 +1,10 @@
 namespace FakeItEasy.Examples
 {
-    using System.Globalization;
-
     public static class CustomArgumentValidators
     {
         public static string IsLongerThan(this IArgumentConstraintManager<string> validations, int length)
         {
-            return validations.Matches(x => x.Length > length, string.Format(CultureInfo.InvariantCulture, "Longer than {0}", length));
+            return validations.Matches(x => x.Length > length, "Longer than {0}", length);

```

Comment:
Well, the "real" `Matches` method takes an `Action<IOutputWriter>`... there is an extension method that takes a string, and another that takes a format string and arguments :wink:
Whether the formatting is done at the call site (using string interpolation) or in the called method doesn't really matter, but I find string interpolation more readable that "classic" string formatting.